tests/queries/0_stateless/02918_sqlite_path_check.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag no-fasttest: Fast tests don't build external libraries (SQLite)
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+function get_exception_message()
+{
+  $CLICKHOUSE_CLIENT --query "$1" |& head -n1 | sed 's/.*DB::Exception: \(.*\) (version.*/\1/g'
```

Comment:
```suggestion
  $CLICKHOUSE_CLIENT --query "$1" |& grep -o "SQLite database file.*$"
```

Thanks @Algunenano for the test, If my understanding is correct, the test is checked against the output in reference file to see if it matches? what do you think about changing this to a simple grep instead ?  Just incase the  response output of the server/client changes in the future? 🤔

Reply:
> Just incase the other parts of the response output from the server to client changes in the future?

The problem with `.*$` is that it will include parts of the exception message that change often, such as the version `(version 23.11.1.1)` (changes every month) or the client address `(from 127.0.0.1:59496)` (changes every run), so we want to avoid printing anything except the main message.

Al alternative could be something like `head -n1 | grep -o "SQLite database file.*(PATH_ACCESS_DENIED)"`. I used sed because I'm more used to it than grep, but I don't mind either way.

Reply:
Ahh let's leave it as is, we can revisit if it ever fails ;) I actually ran into the exact issue you mentioned with grep in the other commit and ended up just use grep with the exact error i'm looking for. 

Thanks for the `|&` trick ... i have been living under the rock and use `2>&1 |` 😱